opendevin/server/listen.py

```diff
@@ -36,6 +45,73 @@
 
 security_scheme = HTTPBearer()
 
+# Determine the project root directory
+PROJECT_ROOT = Path(__file__).parent.parent.parent
```

Comment:
This seems a bit fragile? We don't often move things around, but it does happen. 😅

Reply:
I'll take a look on how to improve this, good point.